AmplifyPlugins/Core/AWSPluginsCore/ServiceConfiguration/AmplifyAWSServiceConfiguration.swift

```diff
@@ -41,4 +41,14 @@ public class AmplifyAWSServiceConfiguration: AWSServiceConfiguration {
     public init(region regionType: AWSRegionType) {
         super.init(region: regionType, credentialsProvider: nil)
     }
+
+    override public init(region regionType: AWSRegionType,
+                         endpoint: AWSEndpoint,
+                         credentialsProvider: AWSCredentialsProvider,
+                         localTestingEnabled: Bool) {
```

Comment:
should this be a feature? ie. release minor version of Amplify with this PR? not sure